datasets/superb/superb.py

```diff
@@ -183,6 +183,64 @@ class Superb(datasets.GeneratorBasedBuilder):
             url="https://www.tensorflow.org/datasets/catalog/speech_commands",
             data_url="http://download.tensorflow.org/data/{filename}",
         ),
+        SuperbConfig(
+            name="ic",
+            description=textwrap.dedent(
+                """\
+            Intent Classification (IC) classifies utterances into predefined classes to determine the intent of
+            speakers. SUPERB uses the Fluent Speech Commands dataset, where each utterance is tagged with three intent
+            labels: action, object, and location. The evaluation metric is accuracy (ACC)."""
+            ),
+            features=datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "speaker_id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "action": datasets.ClassLabel(
+                        names=["activate", "bring", "change language", "deactivate", "decrease", "increase"]
+                    ),
+                    "object": datasets.ClassLabel(
+                        names=[
+                            "Chinese",
+                            "English",
+                            "German",
+                            "Korean",
+                            "heat",
+                            "juice",
+                            "lamp",
+                            "lights",
+                            "music",
+                            "newspaper",
+                            "none",
+                            "shoes",
+                            "socks",
+                            "volume",
+                        ]
+                    ),
+                    "location": datasets.ClassLabel(names=["bedroom", "kitchen", "none", "washroom"]),
+                }
+            ),
+            supervised_keys=None,
+            url="https://fluent.ai/fluent-speech-commands-a-dataset-for-spoken-language-understanding-research/",
+            data_url="http://fluent.ai:2052/jf8398hf30f0381738rucj3828chfdnchs.tar.gz",
+        ),
+        SuperbConfig(
+            name="si",
+            description=textwrap.dedent(
+                """\
+            Speaker Identification (SI) classifies each utterance for its speaker identity as a multi-class
+            classification, where speakers are in the same predefined set for both training and testing. The widely
+            used VoxCeleb1 dataset is adopted, and the evaluation metric is accuracy (ACC)."""
+            ),
+            features=datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "label": datasets.ClassLabel(names=[f"id{i + 10001}" for i in range(1251)]),
```

Comment:
Could you add a comment here on the origin of the `10001` and `1251` numbers please? Just so it's easier to maintain in the future :)